charts/internal/cilium/values.yaml

```diff
@@ -350,6 +350,16 @@ global:
     # interface is the interface to use for encryption
     # interface: eth0
 
+    # ipsec specific configuration
+    # see https://docs.cilium.io/en/stable/security/network/encryption-ipsec/ for further details
+    ipsec:
+      keyId: 3 # the keyId is a number between 1 and 15. `+` sign will be added automatically, so per-tunnel keys will be derived
```

Comment:
We should probably include a bit more detail about this keyId field. In cilium it's the SPI for IPSec encryption:
https://github.com/cilium/cilium/blob/99fec180512f475bd1ceadf7ab12a2ac080535c7/pkg/datapath/linux/ipsec/ipsec_linux.go#L1012